podman/tests/unit/test_000_connection.py

```diff
@@ -0,0 +1,35 @@
+"""Connection unit tests"""
+import unittest
+from podman import ApiConnection
+
+
+# E1101 disabled due to PyCQA/pylint#3157 under py36
+# pylint: disable=R0201,E1101
```

Comment:
Can't we centralize our all these disable in our tox.ini to avoid to spread them all across our code?

By example with flake8 we can use something like:

```
ignore = D100,D101,D102,D103,D104
```

I'm not familliar with pylint but maybe some equivalent exists.

Reply:
There is a .pylintrc where we likely could configure this though I was not certain we want to do so in a global fashion. I will add it there and see if that clears it up. R0201 doesnt' seem like a good check anyway http://pylint-messages.wikidot.com/messages:r0201

But R1101 is specific to the python in the gate container which should probably be addressed in a followup via a fedora 31 -> 33

Reply:
I seen, LGTM, thanks.